### DISCUSSION TITLE
Upgrade from 4.6.x to 4.8.x

### DISCUSSION THREAD

**Discussion by donwlewis** [PowerDNS/pdns#14077] [Q&A] [answered]
I am looking for some guidance in the upgrade path from authoritative 4.6.x to 4.8.x with a PostgreSQL backend.  I attempted this and ran into an error with the schema I believe:

```
Error: GSQLBackend unable to retrieve information about domain 'dc.local': Fatal error during prePQpreparepare: select id,name,master,last_check,notified_serial,type,options,catalog,account from domains where name=$1: ERROR:  column "options" does not exist
LINE 1: ...ct id,name,master,last_check,notified_serial,type,options,ca...
```

**Comment by mnordhoff** [comment, marked as answer]
You should check out the upgrade notes:

https://doc.powerdns.com/authoritative/upgrading.html

There's a `.sql` file you can use to apply the schema changes; it's in the source distribution and there should be a copy installed somewhere like `/usr/share/pdns-backend-pgsql/`.
